.travis.yml

```diff
@@ -2,11 +2,11 @@ sudo: false
 language: ruby
 cache: bundler
 rvm:
-  - 2.1
-  - 2.2
+  - 2.3.4
```

Comment:
Please use `2.2`, `2.3` and `2.4`. 

See https://github.com/dkdeploy/dkdeploy-core/commit/82ff875a1821aa2e189d25bfeac0c9def6534b60#diff-354f30a63fb0907d4ad57269548329e3